microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -77,6 +90,160 @@ def verify_mshvlog_is_active(
+                file=f"{chk_bin_remote_dir}/{chk_bin_tar_file}",
+                dest_dir=chk_bin_extract_dir,
+                gzip=True,
+                sudo=True,
+            )
+            tar.extract(
+                file=f"{chk_loader_remote_dir}/{chk_loader_tar_file}",
+                dest_dir=chk_loader_extract_dir,
+                gzip=True,
+                sudo=True,
+            )
+
+            # Copy CHK bins into test machine
+            copy_tool = node.tools[Cp]
+            copy_tool.copy(
+                src=PurePath(chk_bin_extract_dir) / "Windows" / "System32",
+                dest=PurePath("/boot/efi/Windows"),
+                sudo=True,
+                recur=True,
+            )
+            path = PurePath(chk_loader_extract_dir) / "boot" / "efi" / "lxhvloader.dll"
+            copy_tool.copy(
+                src=path,
+                dest=PurePath("/boot/efi"),
+                sudo=True,
+            )
+
+            # Remove kernel lockdown from grub config
+            node.tools[Sed].substitute(
+                regexp="lockdown=integrity",
+                replacement="",
+                file=grub_config_file,
+                sudo=True,
+            )
+
+            # Add MSHV debug option in chainloader
+            # This is to load hv binaries with debug mode enabled
+            hv_debug_option = "LXHVLOADER_DEBUG=TRUE"
+            grub_config = node.tools[Cat].read(
+                file=grub_config_file,
+                force_run=True,
+                sudo=True,
+            )
+            regex = re.compile(r"(?P<chainloader_cfg>.*chainloader.*)")
+            loader_grub_data = find_group_in_lines(
+                lines=grub_config,
+                pattern=regex,
+                single_line=False,
+            )
+            chainloader_config = loader_grub_data.get("chainloader_cfg", "").strip()
+            err_msg = f"Cannot get chainloader config, got {chainloader_config}"
+            assert chainloader_config, err_msg
+            if hv_debug_option not in chainloader_config:
+                node.tools[Sed].substitute(
+                    regexp="MSHV_SEV_SNP=TRUE",
+                    replacement=f"MSHV_SEV_SNP=TRUE {hv_debug_option}",
+                    file=grub_config_file,
+                    sudo=True,
+                )
+
+            node.tools[Reboot].reboot_and_check_panic(log_path)
+
+            # Trigger HV crash and check if dump is generated
+            hvdbg = node.tools[Ls].path_exists(mshv_debug_sysfs, sudo=True)
+            if not hvdbg:
+                raise LisaException(f"sysfs entry not present: {mshv_debug_sysfs}")
+
+            kdump_util = node.tools[KdumpCheck]
+            kdump_util.kdump_test(
+                log_path=log_path,
+                trigger_kdump_cmd=mshv_crash_command,
+                is_auto=False,
+            )
+        except Exception as e:
+            raise LisaException(f"verify_mshv_crash failed: {e}")
```

Comment:
The new exception error doesn't provide additional information, so the entire except block can be removed.